top_pot.html

```diff
@@ -1,16 +1,16 @@
 <script>
-	function TopPot(location, hoursOfOp, percentEnter, averagePerOrder, hourlyFT, donutsperHour) {
-		this.location = location;
-		this.hoursOfOp = hoursOfOp;
-		this.percentEnter = percentEnter;
-		this.averagePerOrder = averagePerOrder;
-		this.hourlyFT = function () {
-			return (Math.random() * (hourlyFT.max - hourlyFT.min) + 1);
-		};
-		this.donutsPerHour = function () {
-			return this.hourlyFT() * this.percentEnter / 100 * this.averagePerOrder;
-		};
+function TopPot(location, hoursOfOp, percentEnter, averagePerOrder, hourlyFT, donutsperHour) {
```

Comment:
Probably better to set up hourlyMin and hourlyMax as properties on the TopPot object, and then send in those numbers into your object constructor (instead of sending in an object, like you are now)